plugins/proxmox/proxmox_count

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#%# family=auto
+#%# capabilities=autoconf
+
+# Michael Grote
```

Comment:
Please use the *perlpod* documentation header style.

Reply:
do you have an example? i dont knwo what i need from: https://perldoc.perl.org/perlpod

Reply:
The [lvm_ plugin](https://github.com/munin-monitoring/contrib/blob/master/plugins/disk/lvm_) is quite a good (and minimal) example.
Or you could visit the [Plugin Gallery](https://gallery.munin-monitoring.org/), pick a random plugin and click at `Source Code` (at the bottom of the page).
Additionally the [plugin contribution hints](https://github.com/munin-monitoring/contrib/tree/master/plugins/#submit-a-new-plugin) could be a useful read, if you are in the mood.
Thanks!

Reply:
changed it